sample_4.py

```diff
@@ -0,0 +1,15 @@
+class Solution:
+    def fib(self, N: int) -> int:
+        # iterative O(n) time O(1) space
+        if N in [0, 1]:
```

Comment:
Similarly, this one alos, would return wrong numbers if you pass negative values. so, correct it to-
if N <= 0:
    return 0

Reply:
Also added a case for handling negative N.

Reply:
You added another if above this so now you can remove this if statement- 
If N in [0, 1]:
return n